em++.bat

```diff
@@ -1,6 +1,11 @@
 @echo off
-REM Entry point for running python scripts on windows systems.
-REM To modify this file, edit `tools/run_python.bat` and then run
-REM `tools/create_entry_points.py`
+:: Entry point for running python scripts on windows systems.
+:: To modify this file, edit `tools/run_python.bat` and then run
+:: `tools/create_entry_points.py`
```

Comment:
isn't it `REM` though? that's what I remember... but it's been a long time.

Reply:
I noticed this style in emsdk.bat file so I thought I would be consistent with that.

Apparently they are both used for comments with this method being slightly faster:  https://www.robvanderwoude.com/comments.php